occurrence-download/src/main/java/org/gbif/occurrence/download/file/specieslist/SpeciesListCollector.java

```diff
@@ -0,0 +1,43 @@
+package org.gbif.occurrence.download.file.specieslist;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.gbif.dwc.terms.GbifTerm;
+import org.gbif.occurrence.download.hive.DownloadTerms;
+
+public class SpeciesListCollector {
+  
+  private List<Map<String,String>> collectedResults = new ArrayList<>();
+  
+  public List<Map<String, String>> getCollectedResults() {
+    return collectedResults;
+  }
+  
+  public void collectResult(Map<String,String> result) {
```

Comment:
this is just adding occurrence records, it should be aggregating records by taxon key already